internal/app/http/handler.go

```diff
@@ -33,8 +36,16 @@ func NewTmcHandler(handlerService HandlerService, options TmcHandlerOptions) *Tm
 // GetInventory returns the inventory of the catalog
 // (GET /inventory)
 func (h *TmcHandler) GetInventory(w http.ResponseWriter, r *http.Request, params server.GetInventoryParams) {
-
 	filters := convertParams(params)
+	if h.Options.JWTValidation {
```

Comment:
the filters are overriden. When I send a request for inventory with author filter to a namespace I don't have access to, I get the namespace I have access to (so my request filter did not take effect)